fix docs for lr

vincentqb opened the pull request:
Documentation for learning rate does not render well. Fixes #27730.

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28026 fix docs for lr**

Differential Revision: [D17953395](https://our.internmc.facebook.com/intern/diff/D17953395)

vincentqb committed:
fix docs for lr.

vincentqb committed:
Update on "fix docs for lr."

Documentation for learning rate does not render well. #27730.

zou3519 commented:
Could you build the docs locally (via `cd docs && make html && open build/html`)and attach a screenshot of the updated page?

vincentqb committed:
Update on "fix docs for lr"

Documentation for learning rate does not render well. #27730.




[ghstack-poisoned]

vincentqb commented:
Here's the output.

![Screen Shot 2019-10-16 at 11 04 34 AM](https://user-images.githubusercontent.com/3047868/66932552-ce68fb00-f005-11e9-8c27-93b227bd5dab.png)

![Screen Shot 2019-10-16 at 11 09 42 AM](https://user-images.githubusercontent.com/3047868/66932553-ce68fb00-f005-11e9-80b7-e7b2fece3377.png)

Unfortunately, there's not as much white space after the enumerated list as I would like. It's still much better than what was there before :)

zou3519 reviewed (approved):
I'm not sure what to do about the space, but this lgtm
